Sources/Testing/testMacro.js

```diff
@@ -451,6 +459,20 @@ test('Macro methods object tests', (t) => {
     'Getter shall return a copy'
   );
 
+  myTestClass.setMyParameterizedObjectProp({ foo: 12, bar: 89 });
```

Comment:
I don't see a reason to further test setting the entire object, since that functionality should be the same as if we didn't pass in a `params: [...]` option. I will add another prop to the object just to demonstrate that you can set an object with keys outside of the `params` list.